On a Debian system the installed files end up in
/usr/share/xml/xgridfit/ so we have to edit the hard coded
search path accordingly.
http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=625587

--- a/python/xgflib.py
+++ b/python/xgflib.py
@@ -18,7 +18,7 @@ except ImportError:
 VERSION = '2.2'
 JAVA_EXECUTABLE = 'java'
 DISPLAY_DIAGNOSTIC_MESSAGES = False
-XGRIDFIT_DIR = abspath('/usr/local/share/xml/xgridfit/')
+XGRIDFIT_DIR = abspath('/usr/share/xml/xgridfit/')
 XSLT_DIR = abspath(join(XGRIDFIT_DIR, 'lib'))
 UTIL_DIR = abspath(join(XGRIDFIT_DIR, 'utils'))
 SCHEMA_DIR = abspath(join(XGRIDFIT_DIR, 'schemas'))
@@ -1202,7 +1202,7 @@ class Configuration():
                 stderr.write("I don't know a processor or validator " + priority_item + "\n")
                 exit(1)
 
-def run_xgridfit(xgfdir="/usr/local/share/xml/xgridfit/"):
+def run_xgridfit(xgfdir="/usr/share/xml/xgridfit/"):
     XGRIDFIT_DIR = abspath(xgfdir)
     """ Validates and compiles an Xgridfit program. """
     start_time = time()
@@ -1322,7 +1322,7 @@ def run_xgridfit(xgfdir="/usr/local/shar
         print "Elapsed time:", time() - start_time
     return error_free
 
-def run_config(xgfdir="/usr/local/share/xml/xgridfit/"):
+def run_config(xgfdir="/usr/share/xml/xgridfit/"):
     XGRIDFIT_DIR = abspath(xgfdir)
     """ Creates/reads/updates a configuration file. """
     # Parse the command line
@@ -1354,7 +1354,7 @@ def run_config(xgfdir="/usr/local/share/
         return False
     return True
 
-def run_ttx2xgf(xgfdir="/usr/local/share/xml/xgridfit/"):
+def run_ttx2xgf(xgfdir="/usr/share/xml/xgridfit/"):
     XGRIDFIT_DIR = abspath(xgfdir)
     """ Extract TT instructions from a TTX file. """
     start_time = time()
@@ -1445,7 +1445,7 @@ def run_ttx2xgf(xgfdir="/usr/local/share
         print "Elapsed time:", time() - start_time
     return error_free
 
-def run_xgfmerge(xgfdir="/usr/local/share/xml/xgridfit/"):
+def run_xgfmerge(xgfdir="/usr/share/xml/xgridfit/"):
     XGRIDFIT_DIR = abspath(xgfdir)
     start_time = time()
     # Parse the command line
